client/src/pages/Home.tsx

```diff
@@ -1,5 +0,0 @@
-import React from 'react';
-
-export const Home = () => {
-	return <div>Home page</div>;
-};
```

Comment:
I would keep this file to define to which page user should be redirected based on his auth status. If user is authorized, we should redirect him to **HomeSearch**. In other case, redirect him to **LoginAndRegister** component